content/basics/templates/intrinsic-functions/index.md

```diff
@@ -92,17 +92,17 @@ cd cfn101-workshop/code/workspace
 :::
 1. Use the AWS CLI to update the stack. The required parameter `--template-body` have been pre-filled for you. Replace the `ParameterValue` **MyAmiId** with the value you have hardcoded in `resources.yaml` file earlier.
 :::code{language=shell showLineNumbers=false showCopyAction=true}
-aws cloudformation update-stack --stack-name cfn-workshop-ec2 --template-body file://intrinsic-functions.yaml --parameters ParameterKey="AmiID",ParameterValue="MyAmiId"
+aws cloudformation update-stack --stack-name cfn-workshop-resources --template-body file://intrinsic-functions.yaml --parameters ParameterKey="AmiID",ParameterValue="MyAmiId"
 :::
 1. If the `update-stack` command was successfully sent, CloudFormation will return `StackId`.
-:::code{language=shell showLineNumbers=false showCopyAction=true}
-"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-ec2/739fafa0-e4d7-11ed-a000-12d9009553ff"
+:::code{language=shell showLineNumbers=false showCopyAction=false}
+"StackId": "arn:aws:cloudformation:us-east-1:123456789012:stack/cfn-workshop-resources/739fafa0-e4d7-11ed-a000-12d9009553ff"
 :::
  1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** console in a new tab and check if the stack status is **UPDATE_COMPLETE**.
 ::::
 ::::tab{id="local" label="Local development"}
 1. Open the **[AWS CloudFormation](https://console.aws.amazon.com/cloudformation)** link in a new tab and log in to your AWS account.
-1. Click on the stack name, for example **cfn-workshop-ec2**.
+1. Click on the stack name, for example **ccfn-workshop-resources**.
```

Comment:
Typo for cfn

Reply:
Also the image in the results has the stack name as cfn-workshop-ec2

Reply:
good catch...i will upload new image

Reply:
I also need to revert your naming from `*-resources` back to `cfn-workshop-intrinsic-functions` as new naming convention is `cfn-workshop-<lab-name>`

Reply:
Please see the line 26 where we guide customer to work on a new template...

> 1. Open the `intrinsic-functions.yaml` file.

Reply:
Ok the typo was pushed by commits i have reverted. Resolving